examples/catalog/bin/sample_page.dart

```diff
@@ -256,7 +258,7 @@ void main(List<String> args) {
     logError(
       'Error: sample_page.dart failed: $error\n'
       'This sample_page.dart app expects to be run from the examples/catalog directory. '
-      'More information can be found in examples/catalog/README.md.'
+      'More information can be found in examples/catalog/README.md.',
     );
     exit(255);
   }
```

Comment:
these improvements are fine, though it's a bit weird to have the comma there, since there's only one argument and no likelihood of adding a second.